microcosm_flask/fields/query_string_list.py

```diff
@@ -27,3 +27,6 @@ def _deserialize(self, value, attr, obj):
             return attribute_params
         except ValueError:
             raise ValidationError("Invalid query string list argument")
+
+    def _serialize(self, value, attr, obj):
```

Comment:
This is only reasonable if we use `QueryStringList` for serialization of page query string.